app/helpers/admin_helper.rb

```diff
@@ -119,6 +119,23 @@ def back_link
     end
   end
 
+  def format_petition_status(status)
+    formatted = case status
+                when 'open'
+                  'collecting_signatures'
+                when 'referred'
+                  'under_consideration'
+                when 'completed'
+                  'closed'
+                when 'hidden'
+                  'rejected'
+                else
+                  status
+                end
+
+    formatted.humanize
+  end
```

Comment:
The values `referred` and `completed` aren't actual state values so these branches will never be triggered. Probably best to add a `status` method to `Petition`, e.g.

``` ruby
def status
  if closed? && !completed?
    'under_consideration'
  elsif completed?
    'closed'
  elsif open?
    'collecting_signatures'
  elsif hidden?
    'rejected'
  else
    state
  end
end
```

Then you don't need the view helper and you can just change `petition.state.humanize` to `petition.status.humanize`.